tests/test_data_store_api_spatialite.py

```diff
@@ -760,5 +762,24 @@ def test_create_invalid_synonym(self):
             self.store.add_to_synonyms("GeometrySubTypes", "TestName", "TestEntity", "TestChangeID")
 
 
+class FirstConnectionTestCase(TestCase):
+    def test_data_store_fails_at_the_beginning(self):
+        temp_output = StringIO()
+        db_name = os.path.join(FILE_PATH, "__init__.py")
+        with pytest.raises(SystemExit), redirect_stdout(temp_output):
+            DataStore(
+                db_host="",
+                db_username="",
+                db_password="",
+                db_port=0,
+                db_name=db_name,  # Give a file that is not a database
+                db_type="sqlite",
+            )
+        output = temp_output.getvalue()
+        assert "ERROR: SQL error when communicating with database" in output
+        assert "Please check your database file and the config file's database section." in output
+        assert "Current database URL: 'sqlite+pysqlite://:@:0/" in output
```

Comment:
Earlier we set the db_name to an `__init__.py` file - why is the current database URL not showing that name here?

Reply:
The absolute path differs on local machines and on Travis. That's why I didn't include it, but I think it's better if I add one more assert which checks "__init__.py" string

Reply:
Added:
https://github.com/debrief/pepys-import/pull/466/commits/4884893c9ba67bf0bd5b81b5f4531d979c36264f

Reply:
Added:
4884893